### PR TITLE
Improve docs

### PR DESCRIPTION
**Pull request by juandspy**
# Description

Updated the documentation to make the local setup easier.

## Type of change

- Documentation update

## Checklist
* [x] `make before_commit` passes
* [x] updated documentation wherever necessary
* [ ] added or modified tests if necessary
* [ ] updated schemas and validators in [insights-data-schemas](https://github.com/RedHatInsights/insights-data-schemas) in case of input/output change

### CODE REVIEW

**Review by Bee-lee — APPROVED** [review]
LGTM, thanks!

**Review by tisnik — APPROVED** [review]
LGTM

File: docs/local_setup.md [hunk 1]
```diff
@@ -20,6 +20,7 @@ You can download it here <https://gitlab.cee.redhat.com/insights-qe/iqe-ccx-plug
 * edit localhost line in your `/etc/hosts`:  `127.0.0.1       localhost kafka minio`
 * `ingress` image should present on your machine. You can build it locally from this repo
 <https://github.com/RedHatInsights/insights-ingress-go>
+* `ccx-data-pipeline` installed. You can doing it by cloning the [repository](https://github.com/RedHatInsights/ccx-data-pipeline) and running `pip install -r requirements.txt`. Make sure you are using the appropiate version of Python (look at `ccx-data-pipeline` README) and a virtualenv (optional but recommended).
```
Comment (joselsegura) [inline docs/local_setup.md:23]: This repository URL is probably outdated. You need to refer to our internal repository (and I'm not sure if it is a good idea to refer it from a public source)

**Review by joselsegura — COMMENTED** [review]
Just one doubt about referring to a private repository

File: docs/local_setup.md (hunk 1 above)
Comment (juandspy) [reply docs/local_setup.md]: Nice catch! Removing it

**Pull request merged**